src/main/groovy/com/twcable/gradle/sling/SlingServerConfiguration.groovy

```diff
@@ -127,57 +122,65 @@ class SlingServerConfiguration {
         _active = active
     }
 
-    /**
-     * Returns the base URL to control a bundle.
-     */
-    @Nonnull
-    URI getBundleControlBaseUri() {
-        URI base = getBaseUri()
-        new URI(base.scheme, base.userInfo, base.host, base.port, '/system/console/bundles', null, null)
-    }
-
-    /**
-     * Returns the URL to use to do actions on a bundle.
-     */
-    @Nonnull
-    URI getBundleControlUriJson() {
-        URI base = getBundleControlBaseUri()
-        new URI(base.scheme, base.userInfo, base.host, base.port, "${base.path}.json", null, null)
-    }
-
-    /**
-     * Returns the URL to use to do actions on a CQ package.
-     */
-    @Nonnull
-    URI getPackageControlUri() {
-        URI base = getBaseUri()
-        new URI(base.scheme, base.userInfo, base.host, base.port, '/crx/packmgr/service/.json', null, null)
-    }
+//    /**
+//     * Returns the URL to use to do actions on a CQ package.
+//     */
+//    @Nonnull
+//    URI getPackageControlUri() {
+//        URI base = getBaseUri()
+//        new URI(base.scheme, base.userInfo, base.host, base.port, '/crx/packmgr/service/.json', null, null)
+//    }
+//
+//    /**
+//     * Returns the URL to list all the CQ packages.
+//     */
+//    @Nonnull
+//    URI getPackageListUri() {
+//        URI base = getBaseUri()
+//        new URI(base.scheme, base.userInfo, base.host, base.port, '/crx/packmgr/list.jsp', null, null)
+//    }
+//
+//    /**
+//     * Returns the URL to download a CQ package.
+//     */
+//    @Nonnull
+//    URI getPackageDownloadUri() {
+//        URI base = getBaseUri()
+//        new URI(base.scheme, base.userInfo, base.host, base.port, '/crx/packmgr/download.jsp', null, null)
+//    }
```

Comment:
True. Keeping them for the moment as reference for making the needed changes to the package plugin ( https://github.com/TWCable/gradle-plugin-cq-package/tree/bundle-plugin-upgrade )